locales/en-US/app.ftl

```diff
@@ -383,6 +383,12 @@ MarkerContextMenu--copy-url = Copy URL
 MarkerContextMenu--copy-page-url = Copy page URL
 MarkerContextMenu--copy-as-json = Copy as JSON
 
+# This string is used on the marker context menu item when right clicked on any marker.
+# It filters the samples by the marker name.
+# Variables:
+#   $markerName (String) - Name of the marker that is selected.
+MarkerContextMenu--filter-samples-by-marker-name = Filter samples by “<strong>{ $markerName }</strong>” markers
```

Comment:
Is it just me or this could read better with `with`? Also, plural or singular marker?

```
MarkerContextMenu--filter-samples-by-marker-name = Filter samples with “<strong>{ $markerName }</strong>” marker
```

Reply:
After reading the first paragraph of the answer, "instrument" sounds more close to our sentence though. `with` could be more suitable for this case. 🤔 
@julienw @mstange what do you think?

Reply:
After talking about this in the matrix channel, I've thought a bit about this and decided to use `Filter samples within “{ $markerName }” markers` (even though I was opposed to it at first :) )

I wanted to use it because I really wanted a one word alternative to the others.
The closes ones I was considering were `Filter samples taken during "{ $markerName }" markers` and `Filter samples overlapping with “{ $markerName }” markers` but I felt like they are long.

Let me know what you think!